src/WebJobs.Script.WebHost/Management/IMeshInitServiceClient.cs

```diff
@@ -12,5 +14,7 @@ public interface IMeshInitServiceClient
         Task MountBlob(string connectionString, string contentShare, string targetPath);
 
         Task MountFuse(string type, string filePath, string scriptPath);
+
+        Task PublishContainerFunctionExecutionActivities(IEnumerable<ContainerFunctionExecutionActivity> activities);
```

Comment:
nit: rename to PublishContainerActivity? Also, with this addition this interface isn't just about "init". Can we rename to IMeshServiceClient?